frontend/src/app/forms/forms.page.html

```diff
@@ -6,9 +6,15 @@
       </ion-button>
     </ion-buttons>
 
-    <ion-title class="ion-text-center"
-      >Step {{currentMenu.id}}/ {{totalSteps}}</ion-title
-    >
+    <ion-title class="ion-text-center">
+      <ng-container *ngIf="currentStep !== APPLICATION_STEP.Submit">
```

Comment:
According to me template conditionals should be avoided whenever it is easily achievable with TS. We can have property titleText and update it according to the step , as anyways we will be listening to the query change.

Reply:
(This particular example is somewhat "special", so I will comment on the general approach and not this very example.)

I believe that it is less error-prone if you have only one single property in your TS file for a specific "task". For example, in our case, we have the "currentStep". In our TS file, we can "blindly" update this property in any way we want. When it gets updated, then our template will automatically "follow" and display the correct header text. But if we had two properties ("currentStep" and "titleText"), then we would need to be very careful when updating the "currentStep", since we should make sure not to forget to update "titleText". Of course, you can solve this by using some separate methods like "setCurrentStep" or so. But usually, you just want to directly say `this.currentStep = foo`. I see "currentStep" as some sort of "data" or "data model", while I see the title text as a "presentational" thing. The TS file should most of the times hold only "data" items and not really care about the "presentation", while the template does the opposite.

Of course, dividing it like this is not always possible. But in general, I believe that using this approach is less error-prone.

But honestly - it doesn't really matter if we use this approach or the one you suggested. Both are fine. :) I originally wanted to use a TS setter for that, but I guess calling a TS getter in the template is not really ideal (performance wise).